egs/iwslt22_ta/ASR/pruned_transducer_stateless5/train.py

```
@@ -0,0 +1,1301 @@
+        copyfile(src=filename, dst=best_valid_filename)
+
+
+def compute_loss(
+    params: AttributeDict,
+    model: Union[nn.Module, DDP],
+    sp: spm.SentencePieceProcessor,
+    sp_tgt: spm.SentencePieceProcessor,
+    batch: dict,
+    is_training: bool,
+    warmup: float = 1.0,
+    reduction="none",
```

Ruff (0.12.2)

Unused function argument: `reduction`

(ARG001)